src/utils/utils.cpp

```diff
@@ -409,6 +412,66 @@ void EchoInfo(const std::string &message) {
   }
 }
 
+void EchoStats(const std::map<std::string, std::int64_t> &stats) {
+  for (const auto &[key, value] : stats) {
+    if (value == 0) {
+      continue;
+    }
+    if (key == "nodes-created") {
+      std::cout << value << " nodes have been created." << std::endl;
+    }
+    if (key == "nodes-deleted") {
+      std::cout << value << " nodes have been deleted." << std::endl;
+    }
+    if (key == "relationships-created") {
+      std::cout << value << " relationships have been created." << std::endl;
+    }
+    if (key == "relationships-deleted") {
+      std::cout << value << " relationships have been deleted." << std::endl;
+    }
+    if (key == "labels-added") {
+      std::cout << value << " labels have been created." << std::endl;
+    }
+    if (key == "labels-removed") {
+      std::cout << value << " labels have been deleted." << std::endl;
+    }
+    if (key == "properties-set") {
+      std::cout << value << " properties have been updated." << std::endl;
+    }
+  }
+}
+
+void EchoNotification(const std::map<std::string, std::string> &notification) {
+  std::string severity = notification.at("severity");
+  std::transform(severity.begin(), severity.end(), severity.begin(), ::toupper);
+  if (severity == "WARNING") {
+#ifdef _WIN32
+    HANDLE hConsole;
+    WORD original_console_attr;
+    CONSOLE_SCREEN_BUFFER_INFO csbi;
+
+    hConsole = GetStdHandle(STD_OUTPUT_HANDLE);
+    if (!GetConsoleScreenBufferInfo(hConsole, &csbi)) {
+      std::cout << severity << ": ";
+    }
+
+    original_console_attr = csbi.wAttributes;
+    FlushConsoleInputBuffer(hConsole);
+    SetConsoleTextAttribute(hConsole, FOREGROUND_RED | FOREGROUND_GREEN |
+                                          FOREGROUND_INTENSITY);
+
+    std::cout << severity << ":";
+    SetConsoleTextAttribute(hConsole, original_console_attr);
+    std::cout << " ";
+#else  /* _WIN32 */
+    std::cout << "\033[1;33m" << severity << ": \033[0m";
+#endif /* _WIN32 */
+    std::cout << notification.at("code") << std::endl;
```

Comment:
```suggestion
    std::cout << notification.at("code") << std::endl;
```
Can be extracted because it's printed out regardless.
Only printing out of the severity needs to stay.

Also, why are you printing out the code and not the message?

Reply:
So you would print out severity, code and title? e.g. that would be like
INFO: CreateIndex - Create Index on label <label_name> on properties <properties>.
Is that ok?

Reply:
I wouldn't print the code at all, just the text. What's the value to the user seeing the code?

Reply:
Every code is short description of what happened, and I feel like every single one is self-explanatory

Reply:
But you are right the code is redundant if it's displayed with a message.